content/SCALE/GettingStarted/Configure/UIConfigurationSCALE.md

```diff
@@ -45,11 +45,10 @@ To apply the license information, go to the **System Settings > General** screen
 This content applies to TrueNAS Enterprise customers only.
 
 TrueNAS SCALE Enterprise customers with Silver or Gold Coverage support contracts have the option to configure proactive support.
-{{< expand "Configuring Proactive Support" "v" >}}
-Customers with appropriate support contracts can configure **[Proactive Support]({{< relref "GetSupportSCALE.md" >}})** after they apply their system license, and after acknowledging and signing the End User License Agreement (EULA).
+
+Customers with appropriate support contracts can configure **[Proactive Support]({{< relref "AddLicenseProactiveSupport.md" >}})** after they apply their system license, and after acknowledging and signing the End User License Agreement (EULA).
```

Comment:
Line 239: 
**[Updating SCALE]
Remove the double asterisk before [Updataing SCALE]

Reply:
found an incomplete thought! good catch. fixed it in the next commit to this PR.